### PR TITLE
Embedding video link into sensitive data overview intro

### PR DESCRIPTION
**Pull request by EKaroune**
### Summary

Testing to see if the video embeds properly

Fixes #<NUM>

Added script for embedding a video from youtube

### List of changes proposed in this PR (pull-request)

- [x] Has it embedding properly?

### What should a reviewer concentrate their feedback on?

- [x] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/project-design/sdp/types.md [hunk 1]
```diff
@@ -11,3 +11,9 @@ These are versions of inner source working in which open collaboration and open
 Therefore each research project needs to find the best solution for their needs.
 
 The following sub-chapters give an overview of the types of sensitive data and metadata that need to be identified in datasets and therefore handled securely in research projects.
+
+Or you could watch this video below that has been created by ELIXIR-UK as an introduction to sensitive data.
```
Comment (malvikasharan) [inline book/website/project-design/sdp/types.md:15]:
```suggestion
Here is a short video introducing sensitive data by ELIXIR-UK.
```

Comment (malvikasharan) [reply book/website/project-design/sdp/types.md:15]: Looks good to me, https://deploy-preview-2850--the-turing-way.netlify.app/project-design/sdp/types.html.

**Review by Arielle-Bennett — APPROVED** [review]
👍🏻

**Comment by EKaroune** [comment]
Thanks @malvikasharan  and @Arielle-Bennett for the reviews

**Pull request merged**